src/renderer/components/wallet/txs/receive/Receive.tsx

```diff
@@ -79,15 +101,11 @@ export const Receive: React.FC<Props> = (props): JSX.Element => {
             </Styled.QRWrapper>
           </Styled.Card>
           <Styled.Div>
-            {isDesktopView ? (
-              <label htmlFor="clipboard-btn">
-                <Styled.Address size="large">{addressLabel}</Styled.Address>
-              </label>
-            ) : (
-              <Styled.AddressWrapper htmlFor="clipboard-btn">
-                <Styled.Address size="large">{addressLabel}</Styled.Address>
-              </Styled.AddressWrapper>
-            )}
+            <label htmlFor="clipboard-btn">
+              <Styled.Address size="large">
+                {isDesktopView ? addressLabel : truncateAddress(addressLabel, chain as Chain, network)}
```

Comment:
```
chain as Chain
```
Don't cheat! If you don't have a network and chain, but you do need it to run `truncateAddress`, then you can't use `truncateAddress`. Or re-think about implementation of props for `Receive` component or of `truncateAddress` - but don't cheat ;)